pkg/telemetry/test/telemetry_service_test.go

```diff
@@ -155,7 +155,7 @@ func Test_OnDownStreamRTCP(t *testing.T) {
 	require.Equal(t, 1, int(stats[0].NackCount))
 	require.Equal(t, 1, int(stats[0].PliCount))
 	require.Equal(t, 1, int(stats[0].FirCount))
-	require.Equal(t, 4, int(stats[0].Delay))      // max of delay, see list of rtcp.ReceptionReport above
+	require.Equal(t, 0, int(stats[0].Delay))
```

Comment:
Looks like need some extra test util to ensure there is some valid data. Makes it harder with the use of current clock. Maybe a TODO?

Reply:
added TODO